tests/integration/test_integration_watcher.py

```diff
@@ -1,20 +1,32 @@
 from fastapi import FastAPI
 from starlette.testclient import TestClient
 
+from antarest.core.tasks.model import TaskStatus
 
-def test_integration_xpansion(app: FastAPI, tmp_path: str):
-    client = TestClient(app, raise_server_exceptions=False)
+
+def test_integration_watcher(app: FastAPI, tmp_path: str):
+    client = TestClient(app)
     res = client.post(
         "/v1/login", json={"username": "admin", "password": "admin"}
     )
     admin_credentials = res.json()
     headers = {"Authorization": f'Bearer {admin_credentials["access_token"]}'}
 
-    client.post(
-        f"/v1/watcher/_scan",
-        headers=headers,
-    )
-    client.post(
-        f"/v1/watcher/_scan?path=/tmp",
+    res = client.post("/v1/watcher/_scan", headers=headers)
+    assert res.status_code == 422
+    assert res.json() == {
+        "description": "field required",
+        "exception": "RequestValidationError",
+        "body": None,
+    }
+
+    task_id = client.post(
+        "/v1/watcher/_scan?path=/default",
         headers=headers,
-    )
+    ).json()
+
+    # asserts that scanning default workspace doesn't fail
+    res = client.get(
+        f"v1/tasks/{task_id}?wait_for_completion=true", headers=headers
+    ).json()
+    assert res["status"] == TaskStatus.COMPLETED.value
```

Comment:
OK, but the scanning of the `default` is prohibited, so we need to check that the task failed.